### ISSUE TITLE
0.3.0-alpha.2 changelog

### ISSUE THREAD

**Issue by sommo** [KraigM/homebridge-harmonyhub#106]
Hi,
via npm update:
npm -g outdated
Package                Current         Wanted         Latest  Location
homebridge-harmonyhub    0.2.0  0.3.0-alpha.2  0.3.0-alpha.2

inform me that there is an update for this plugin, could we know what's the changelog?
Thank you

**Comment by BetoRn** [comment]
From what i've seen there's now a speaker device, wich can control volume for the active activity
a changelog would be great!

**Comment by sommo** [comment]
Have tuo or Anyone else installed and tested? So it will add a new device in homekit?

Inviato da iPhone

**Comment by tfonfara** [comment]
I'm testing it since 3 days now and at least I can say that it improves stability so far 👍 (0.2.0 version used to crash/hang up after a couple of hours and homebridge got unavailable in homekit)

**Comment by patrickbusch** [comment]
how did you get the volume control? I updated, but do not see it. Is it an additional setting?

**Comment by BetoRn** [comment]
Speaker appears on el gato eve app possibly other 3rd party apps too

**Comment by sullimander** [comment]
Yeah, I can see the speaker in the elgato eve app too, but can't figure out how to get Siri to control it, or for it to show up in my Homekit app. Anyone figure this out?

**Comment by ToddGreenfield** [comment]
Try creating a scene in Eve to set the volume. Then call the scene with siri.
